tests/terraform/runner/resources/example/example.tf

```diff
@@ -344,6 +344,43 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
     viewer_protocol_policy = "redirect-to-https"
   }
 
+dynamic "ordered_cache_behavior" {
+    for_each = var.ordered_cache
```

Comment:
@arielkru What was throwing me off is the indent here - missing one tab. Otherwise - @schosterbarak this is an example of the string block because `var.ordered_cache` is null, so this is parsed by checkov as a string and not as a dict. Even though we have the `content` block.